app/controllers/feedback_controller.rb

```diff
@@ -8,7 +8,10 @@ def create
     unless form_values.empty?
       FeedbackMailer.feedback(current_user, form_values).deliver_later
```

Comment:
When we settle on a support service, we may want to use an API rather than going through a emails, which are harder to capture errors for.